netutils/utils.py

```diff
@@ -1,7 +1,11 @@
 """Utilities for the netutils library."""
```

Comment:
I don't think this is the right file for this. We should have something like parsers or similar.

Reply:
I almost think something like `lib_helpers` or something would be good. Thoughts?